heron/executor/src/python/heron_executor.py

```diff
@@ -585,6 +606,44 @@ def _get_streaming_processes(self):
 
     return retval
 
+  def _get_ckptmgr_process(self):
+    ''' Get the command to start the checkpoint manager process'''
+
+    ckptmgr_main_class = 'com.twitter.heron.ckptmgr.CheckpointManager'
+
+    ckptmgr_cmd = [os.path.join(self.heron_java_home, "bin/java"),
+                   '-Xmx512M',
```

Comment:
Can Xmx be configurable? Since all state transfers through the checkpoint manager, it could possibly need more memory.